config/locales/simple_form.en.yml

```diff
@@ -53,6 +53,7 @@ en:
       profile:
         address: "Street Address for Pickup"
         notes: "Pickup Notes"
+        notes: "Phone Number"
```

Comment:
I think explicit is probably better so we avoid the user wondering what we be using their number for.